collect_app/src/main/java/org/odk/collect/android/geo/GoogleMapFragment.java

```diff
@@ -194,7 +192,7 @@ public void onSaveInstanceState(@NonNull Bundle outState) {
     }
 
     @Override public void onDestroy() {
-        MapsMarkerCache.clearCache();
+        BitmapDescriptorCache.clearCache();
```

Comment:
You'll still want to clear the `MapsMarkerCache` I think? `BitmapDescriptorCache` ends up sticking bitmaps in there. Maybe `BitmapDescriptorCache#clearCache()` should clear the `MapsMarkerCache`?

We probably want a more general cache solution that could simplify these kinds of interactions (and possibly event interact with Android Lifecycle), but I think waiting for a third object is worth it before trying to create that kind of abstraction.

Reply:
Right, fixed.